aws/sdk/integration-tests/ec2/tests/paginators.rs

```diff
@@ -19,28 +18,28 @@ fn stub_config(http_client: impl HttpClient + 'static) -> Config {
         .build()
 }
 
+fn validate_query_string(expected: &str, actual: &str) {
+    let expected = expected.split('&').collect::<HashSet<&str>>();
+    let actual = actual.split('&').collect::<HashSet<&str>>();
+    assert_eq!(expected, actual);
```

Comment:
I think this would miss if the same param appeared in  multiple times in one and not in the other? Shouldn’t actually impact the behavior, but might be worth asserting on the matching lengths as well to catch duplications?

Reply:
Yeah, that crossed my mind. Maybe I should collect into sorted Vecs and compare them.

Reply:
1dbf37a added assertion on matching lengths